serverless/tensorflow/matterport/mask_rcnn/nuclio/main.py

```diff
@@ -19,6 +19,7 @@ def init_context(context):
 
     context.logger.info("Init context...100%")
 
+
```

Comment:
I don't think that you need to change the file at all. Could you please revert changes?